<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="age_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="male_radio">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="female_radio">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CARS score</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Observations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>440</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Input patient Information</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="age_input">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Leave the field empty if the patient &lt;/p&gt;&lt;p&gt;has not been diagnosed with ASD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Diagnosis</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="TC_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>TC</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="TS_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>TS</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="not_diagnosed_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Not diagnosed yet</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="cars_score">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
